apis/types/capability.go

```diff
@@ -134,6 +134,7 @@ type Parameter struct {
 	Required bool        `json:"required,omitempty"`
 	Default  interface{} `json:"default,omitempty"`
 	Usage    string      `json:"usage,omitempty"`
+	Ignore   string      `json:"ignore,omitempty"`
```

Comment:
Will changing the type to bool make it more clear? All annotation are in `key=value` format. And the default value is `false`.
`// +ignore If addRevisionLabel is true, the appRevision label will be added to the underlying pods` will be 
```
// +ignore=true
 // +usage=If addRevisionLabel is true, the appRevision label will be added to the underlying pods

@wonderflow Do you think it's a good idea?
```

Reply:
mark `+ignore` here means we should ignore it in `vela show`, to avoid confusion, I think the example should in the following format

```
// +ignore
// If addRevisionLabel is true, the appRevision label will be added to the underlying pods
```

Reply:
While I also agree @zzxwill  if we want to make the syntax to be simple and consistent.

```
// +ignore=true
// +usage=If addRevisionLabel is true, the appRevision label will be added to the underlying pods
```

Also works for me, but I think the default value is true. Nobody will mark `+ignore=false`.

Reply:
+1

Reply:
Oh, the default should be `false`. If the description part is spilt into `+usage`, just leaving `+ignore` as string type is okay with me.